stickyuploads/storage.py

```diff
@@ -12,4 +15,18 @@ def __init__(self):
 
     def url(self, name):
         """Files are not accessable via url."""
-        raise NotImplementedError()
\ No newline at end of file
+        raise NotImplementedError()
+
+    def get_available_name(self, name, max_length=None):
+        """Return path to name placed in two random directories"""
+        name = os.path.join(
```

Comment:
The original issue was meant to address creating unique directories for each upload to prevent conflicts and the resulting file renames. `mkdtemp` would a trusted way to do that. This optimizes the number of files/directories created over the security and uniqueness provided using the standard lib. You haven't convinced me that's a good trade off.